include/sway/input/input-manager.h

```diff
@@ -44,9 +44,13 @@ void input_manager_configure_xcursor(void);
 
 void input_manager_apply_input_config(struct input_config *input_config);
 
+void input_manager_configure_input(struct sway_input_device *input_device);
```

Comment:
It doesn't look like this is being used outside of `input-manager.c`. We generally try to keep all functions static until they are used elsewhere.

Reply:
For symmetry, I based this off `input_manager_reset_input` below which is also exported despite not being used elsewhere. I've dropped this one from the header, though.

Reply:
I think it may have been used at some point and just never converted to static. I could be wrong and it was just an oversight though. Either way, thanks. We do try to follow that, but it's not a strict requirement.